### PR TITLE
add MeiliSearch informations

### PR DESCRIPTION
**Pull request by fharper**
Descriptions, keywords, logos

### CODE REVIEW

**Comment by fharper** [comment]
@tpayet & @qdequele : I added you as reviewers so you can fine tune the description and keywords.

I know we have some logos in the integrations repo, and on a Notion page, but I thought it would be nice to have them ALL in one place, and in a repo so we can easily track the history.

**Comment by fharper** [comment]
Actually, @react-learner can review it! If the founders wants to change something, they can do a PR :D

File: assets/information.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  - [MeiliSearch Logos](#meilisearch)
+  - [Software Developement Kits (SDKs)](#software-developement-kits-sdks)
+  - [Framework Logos](#frameworks)
+  - [Integrations Logos](#integrations)
+  - [Cloud Logos](#cloud)
+
+## Product Description
+
+### Short
+MeiliSearch is an open source, blazingly fast and hyper relevant search-engine.
```
Comment (react-learner) [inline assets/information.md:15]:
```suggestion
MeiliSearch is an open source, blazingly fast, and hyper relevant search engine.
```

File: assets/information.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+### Long
+MeiliSearch is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, and synonyms are provided out-of-the-box.
+
+### Keywords
+Search Engine, Open Source, Rust, Fast
+
+## Logos
+Here a list of official logos we use in vectorial format.
```
Comment (react-learner) [inline assets/information.md:24]:
```suggestion
Here is a list of official logos we use in vectorial format.
```

**Review by react-learner — APPROVED** [review]
LGTM.

**Pull request merged**